translations/da_DK.ts

```diff
@@ -822,7 +822,7 @@
     <name>NinjamPanel</name>
     <message>
         <source>Sync with %1</source>
-        <extracomment>The &apos;%1&apos; marker will be replaced by the host name when Jamtaba is running.</extracomment>
+        <extracomment>&apos;%1&apos; Markøren vil blive erstattet af værtsnavnet, når Jamtaba kører.</extracomment>
```

Comment:
@Hitrate , here in line 825 you translated the "extracomment". The extracomment is not translatable, is just a tip for translators to understand the context.

Can you rollback this change in extracomment?